src/plugins/console/public/lib/mappings/mappings.ts

```diff
@@ -52,15 +97,15 @@ export function expandAliases(indicesOrAliases) {
     indicesOrAliases = [indicesOrAliases];
   }
 
-  indicesOrAliases = indicesOrAliases.map((iOrA) => {
+  const mappedIndicesOrAliases = indicesOrAliases.map((iOrA) => {
     if (perAliasIndices[iOrA]) {
       return perAliasIndices[iOrA];
     }
     return [iOrA];
   });
-  let ret = [].concat.apply([], indicesOrAliases);
+  let ret: string[] = ([] as string[]).concat(...mappedIndicesOrAliases);
```

Comment:
This change was made for better typing, the return array of function remains same, because the array is flattened before it. The behavior is unchanged